File: atcoder/abc077A_rotation.nim
# unihernandez22
# https://atcoder.jp/contests/abc077/tasks/abc077_a
# implementation

import strutils, algorithm

let
  a = stdin.readLine.strip
  b = stdin.readLine.strip

if a.reversed.join == b:
  echo "YES"
else:
  echo "NO"

File: atcoder/abc077A_rotation.ml
(* vicfred
 * https://atcoder.jp/contests/abc077/tasks/abc077_a
 * implementation
 * *)
Scanf.scanf "%s\n%s\n" ( fun a b ->
    if a.[0] = b.[2] && a.[1] = b.[1] && a.[2] = b.[0] then "YES"
    else "NO"
) |> Printf.printf "%s\n"

